src/base/utils/io.h

```diff
@@ -88,6 +88,7 @@ namespace Utils::IO
         enum Code
         {
             NotExist,
+            ErrorReading,
```

Comment:
I wouldn't still repeat `Error` word as well as `Read` (`ReadError::ErrorReading` looks ugly) and use something like `Failed`.